app/serializers/group_serializer.rb

```diff
@@ -57,4 +57,8 @@ def has_discussions
   def has_multiple_admins
     object.admins.count > 1
   end
+
+  def cover_photo
```

Comment:
Yeah, heh. Had a similar thought. There's lots of cases where we load a
group and don't care about the cover photo, but I think an 'includes'
default_scope is way less likely to come back and bite us (or it will take
longer to come back and bite us, heh) than a 'where' default_scope

On Tue, Oct 20, 2015 at 11:46 AM, Robert Guthrie <notifications@github.com>
wrote:

> In app/serializers/group_serializer.rb
> <https://github.com/loomio/loomio/pull/2590#discussion_r42469352>:
>
> > @@ -57,4 +57,8 @@ def has_discussions
> >    def has_multiple_admins
> >      object.admins.count > 1
> >    end
> > +
> > +  def cover_photo
>
> So... how crazy do you think it would be to put something like
>
> default_scope { includes(:default_cover_photo) }
>
> into group.rb
>
> I've always ended up regretting using default_scope.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2590/files#r42469352>.
>